### DISCUSSION TITLE
Add RTL support to Tabs

### DISCUSSION THREAD

**Discussion by mhmdev64** [tailwindlabs/headlessui#3043] [Ideas]
Hi there,

I hope this message finds you well. I've been thoroughly enjoying using the Headless UI library for my projects, and I recently came across a couple of points I'd like to bring to your attention.

**Bug Report**:
While working with the tab, and navigate with keyboard between tabs is not works as expected in RTL.

**Feature Request**:
I would like to suggest the addition of RTL support for the Tab component. This enhancement would greatly improve the usability of the library for projects that require support for languages written from right to left.

Thank you for your continuous efforts in maintaining and improving the Headless UI library. I appreciate your time and consideration.

**Comment by RobinMalfait** [comment]
Hey!

Do you mind sharing what kinds of features you are missing or the behaviour you are looking for that currently doesn't play well with the Tabs component when using languages that are written from right to left?

Would love to learn more about this so that we can make your experience better.

**Comment by mhmdev64** [comment]
Horizontal keyboard navigation in LTR is good but must be opposite in RTL.

**Reply by RobinMalfait** [reply]
I see what you mean, thanks!

I added it to our list of ideas to support. I can't promise you anything right now or give you an ETA.

**Reply by mostafa-jahani** [reply]
You can change the property right-0 to left-0 in ComboboxButton to solve this problem.
